lib/floe/workflow/state.rb

```diff
@@ -92,6 +92,42 @@ def ready?
       def finished?
         context.state.key?("FinishedTime")
       end
+
+      def wait_until
+        context.wait_until
+      end
+
+      private
+
+      # Use a payload value or hardcoded path.
+      #
+      # @param value      value hardcoded from the payload
+      # @param value_path path to the value
+      # @param block      method use for converting the path lookup (string) to the proper datatype
+      # @returns value derived from hardcoded value or path
+      def value_or_path(value, value_path)
+        if value_path
+          value = value_path.value(context, context.input)
```

Comment:
please double check this line.

Do we want to use `context, context.input`
Or is it better to pass in the context and input and then we will pass in the input after the input_path transforms?